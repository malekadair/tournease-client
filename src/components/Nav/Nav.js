import React, { Component } from "react";
import "../App/App.css";
import { Link } from "react-router-dom";
class Nav extends Component {
  render() {
    return (
      <nav>
        <Link to="/">
          <h2>TournEase</h2>
        </Link>
        <ul className="nav-links">
          <li>
            <Link to="/">Find Tournaments</Link>
          </li>
          <li>
            <Link to="/create">Create</Link>
          </li>
          <li>
            <Link to="/">Log Out</Link>
          </li>
        </ul>
      </nav>
    );
  }
}

export default Nav;
