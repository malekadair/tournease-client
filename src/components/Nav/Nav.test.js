import React from "react";
import toJson from "enzyme-to-json";
import Nav from "./Nav";

describe(`Nav component`, () => {
  const props = {
    id: "a",
    name: "test-class-name"
  };

  it("renders a .Nav by default", () => {
    const wrapper = <Nav />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("renders the Nav given props", () => {
    const wrapper = <Nav {...props} />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
