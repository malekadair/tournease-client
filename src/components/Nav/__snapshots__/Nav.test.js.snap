// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component renders a .Nav by default 1`] = `undefined`;

exports[`Nav component renders the Nav given props 1`] = `undefined`;
